src/components/FormButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import { COLOR } from '../utils/color';
+import type { ButtonHTMLAttributes } from 'react';
+
+interface FormButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  children?: ReactNode;
+  type: 'submit' | 'button';
+}
+
+const FormButton = ({ type, children, ...props }: FormButtonProps) => {
+  return (
+    <StyledButton type={type} style={{ ...props.style }} {...props}>
+      {children}
+    </StyledButton>
+  );
+};
+
+export default FormButton;
+
+const StyledButton = styled.button`
+  width: 100%;
+  font-family: 'Inter';
```

Comment:
```suggestion
  font-family: 'Inter', sans-serif;
```
이렇게 수정해주시면 좋을 것 같아요~